Automattic/jetpack
# Blogroll: change block icon

This PR fixes the blogroll icon, to be according to the new design.

p9Jlb4-8nF-p2#the-icon

Related diff: D119739-code

Fixes #
https://github.com/Automattic/wp-calypso/issues/80907

## Proposed changes:

* Update the icon for the blogroll block

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the quick inserter and search for blogroll
* The icon for the block should match the design
